public/js/search.js

```diff
@@ -491,6 +491,24 @@ var Databases = React.createClass({
         this.setState({type: type});
     },
 
+    handleToggle: function (type) {       
```

Comment:
A much simpler way exists:
```
$(`.${type} .database`).click();
```

Reply:
Further,

1. When nucleotide databases are selected, 'Toggle all' button for protein databases should be disabed and vice versa.
2. 'Toggle all' button should only be shown when there are two or more databases.